examples/bert/config_data.py

```diff
@@ -7,7 +7,7 @@
 max_batch_tokens = 128
 
 train_batch_size = 32
-max_train_epoch = 5
+max_train_epoch = 3
```

Comment:
will this affect other code's results? E.g., `bert_classifier_main.py`

Reply:
Reverted to `max_train_epoch = 5` in e175fa6. I was `max_train_epoch = 3` because it takes relatively less time to finish.